pyproject.toml

```diff
@@ -86,24 +84,36 @@ Home-page = "https://scanpy.org"
 scanpy = "scanpy.cli:console_main"
 
 [project.optional-dependencies]
-test = [
+test-min = [
     "pytest>=4.4",
     "pytest-nunit",
-    "dask[array]!=2.17.0",
-    "fsspec",
-    "zappy",
-    "zarr",
     "profimp",
-    # Test the metadata while this exists: https://github.com/takluyver/flit/issues/387
-    "flit_core",
+]
+test = [
+    "scanpy[test-min]",
+    # Optional but important dependencies
+    "scanpy[leiden]",
+    "zarr",
+]
```

Comment:
OK so what’s your plan with this?

- I just guessed the set of “optional but important” tests. Which ones should it be?
- Do we want to check if our set of “optional but important” tests aren’t skipped? If so, how?

Reply:
This looks fine for now.

I thought about it more and am not sure of the best approach so I'm good to merge this once the tests pass.